test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
I tracked it to https://github.com/JuliaLang/julia/blob/2b6406870da57635900df450fe1f9499bb989ef4/test/runtests.jl which is adding and removing workers only for unix. https://github.com/JuliaLang/julia/commit/89c8cce10a6777a5e87480448b4c757218bd5f02 actually removed the `@unix_only` but somehow got lost...